JSDemos/Demos/VectorMap/DynamicViewport/description.md

```diff
@@ -1 +1,5 @@
-This demo illustrates how to&nbsp;change the map&rsquo;s viewport from code. Choose a&nbsp;continent using the drop-down menu under the map. In&nbsp;response to&nbsp;that, the map changes the visible area so&nbsp;that the chosen continent is&nbsp;displayed optimally. This is&nbsp;performed in&nbsp;code by&nbsp;calling the **viewport(viewportCoordinates)** method. After that, callback functions assigned to&nbsp;the **centerChanged** and **zoomFactorChanged** properties are triggered. These functions place the current values of&nbsp;the map&rsquo;s center and zoom factor in&nbsp;the text boxes under the map.
\ No newline at end of file
+This demo illustrates how to change the map viewport. Use the drop-down menu under the map to choose a continent. The map changes the visible area so that the chosen continent is displayed optimally as a result.
```

Comment:
```suggestion
This demo illustrates how to change the map's viewport. Use the drop-down menu under the map to choose a continent and change the visible area so that the chosen continent is displayed optimally.
```